libgeopmd/src/LevelZeroDevicePool.cpp

```diff
@@ -302,8 +318,10 @@ namespace geopm
         dev_subdev_idx_pair = subdevice_device_conversion(domain_idx);
         check_domain_exists(m_levelzero.engine_domain_count(dev_subdev_idx_pair.first, l0_domain),
                             __func__, __LINE__);
-        return m_levelzero.active_time(dev_subdev_idx_pair.first, l0_domain,
-                                       dev_subdev_idx_pair.second);
+        uint64_t active_time = m_levelzero.active_time(dev_subdev_idx_pair.first, l0_domain,
+                                                       dev_subdev_idx_pair.second);
+
+        return convert_active_time(active_time, m_active_time_last[domain_idx], m_active_time_rollover[domain_idx]);
```

Comment:
There needs to be another dimesion to m_active_time_lastand m_active_time_rollover: one for each l0_domain.

Reply:
https://github.com/geopm/geopm/pull/3615/commits/bce35cbab30dc36fc5d24d4fa8a0d58ea11efcee